src/Tribe/Admin/Notices.php

```diff
@@ -237,4 +238,44 @@ public function maybe_display_plus_commerce_notice() {
 			tribe_notice( "event-tickets-plus-missing-{$provider}-support", $message, 'dismiss=1&type=warning' );
 		}
 	}
+
+	/**
+	 * Display notices for unsupported currencies.
+	 *
+	 * @return void
+	 */
+	public function maybe_display_unsupported_currency_notice() {
+		// Get currencies and notice texts.
+		$unsupported_currency = [
+			'HRK' => [
+				'heading' => __( 'Tickets Commerce is now selling with Euro', 'event-tickets' ),
+				'message' => __( 'From the 1st of January 2023, the euro became the official currency for Croatia. We have removed the Croatian kuna from our currency settings and updated your settings to start selling with Euro.', 'event-tickets' ),
+				'updated' => 'EUR'
+			],
+		];
+
+		// Get currency code option
+		$currency = tribe_get_option( 'tickets-commerce-currency-code' );
```

Comment:
We have the option name living in a constant, we could probably consume it from there: 
- https://github.com/the-events-calendar/event-tickets/blob/master/src/Tickets/Commerce/Utils/Currency.php#L25
- https://github.com/the-events-calendar/event-tickets/blob/master/src/Tickets/Commerce/Settings.php#L54

Reply:
@juanfra Thanks for this.